components/scripts/lib/summary.sh

```diff
@@ -181,10 +181,19 @@ print_performance_characteristics() {
 
 print_performance_characteristics_header() {
   print_bl
-  info "Performance Characteristics"
+  info "Performance Characteristics ($(analyzing_the_results_link))"
   info "---------------------------"
 }
 
+# TODO update these links
+analyzing_the_results_link() {
+  if [[ "${BUILD_TOOL}" == "Maven" ]]; then
+    echo "https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/erichaagdev/performance-characteristics-explained/Maven.md#performance-characteristics"
```

Comment:
todo: update these links

It would be a nice improvement if these links did not always point to `main`, but to the tag for released versions. However, that is not something we do anywhere else in the code. 

https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/e6eae99d02a18a91dcc5a0b01c98b28cf30978a0/components/scripts/lib/interactive-mode.sh#L48

If we wanted to do that, it should probably be a future enhancement (subsequent PR).

Reply:
Yes, I would not do this for now. It's not something we don't do for our GE plugin/extension, neither.
But, you are right in that it would be an alternative.